modules/appeals_api/app/controllers/appeals_api/metadata_controller.rb

```diff
@@ -0,0 +1,48 @@
+# frozen_string_literal: true
+
+require 'appeals_api/health_checker'
+
+module AppealsApi
+  class MetadataController < ::ApplicationController
+    skip_before_action :verify_authenticity_token
+    skip_after_action :set_csrf_header
+    skip_before_action(:authenticate)
+
+    def healthcheck
+      if AppealsApi::HealthChecker.services_are_healthy?
+        render json: healthy_service_response
+      else
+        render json: unhealthy_service_response,
+               status: :service_unavailable
+      end
+    end
+
+    private
+
+    def healthy_service_response
+      {
+        data: {
+          id: 'appeals_healthcheck',
```

Comment:
i've been using `id: nil`, personally, for dummy resources. Idk if there's been any agreement on what should be used when something is not a db-backed resource